rasa/core/policies/unexpected_intent_policy.py

```diff
@@ -285,13 +302,14 @@ def __init__(
             if self.label_quantiles
             else {}
         )
-        self.ignore_intent_list = self.config[IGNORE_INTENTS_LIST]
+        self.ignore_intent_list = config.get(IGNORE_INTENTS_LIST)
```

Comment:
why did we need to change this?

Reply:
That's right, it's in fact not really needed.

Reply:
Updated it (basically reverted back)